src/client/components/DiscussItem.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+import { TeamContext } from '../lib/teamContext';
+
+export function DiscussItem(props) {
+  function getReplyUrl(serverUrl, diaryChannel, title, authors) {
+    let url = `${serverUrl}/create-thread?`;
+    let paramCount = 0;
+    if (diaryChannel) {
+      paramCount++;
+      url = `${url}parentChannel=${diaryChannel}`;
+    }
+
+    if (title) {
+      paramCount++;
+      url = `${url}${paramCount > 1 ? '&' : ''}message=>%20${title}%0A%0A${authors && authors.length > 0 ? `${authors.map(author => `%40${author}`)}%20` : ''}`;
+    }
```

Comment:
How do you think about collecting parameters in an array first and then appending `'?' + params.join('&')` - would be more clear in my opinion

Reply:
adjusted. No clue why Github still shows old code...